examples/gocean/eg6/Makefile

```diff
@@ -0,0 +1,45 @@
+F90 ?= gfortran
```

Comment:
Copyright/license and please could you make the NETCDF include and lib directories configurable.

Reply:
Again, I am using ``nc-config``.